src/pages/Dashboard/Portfolio.tsx

```diff
@@ -10,7 +10,7 @@ import { DashboardChartSkeleton } from './components/DashboardChartSkeleton'
 import { Display } from '@/components/Display'
 import { Text } from '@/components/Text'
 
-const cardBodyPx = { base: 4, md: 2 }
+const cardBodyPx = 2
```

Comment:
nitpick: no need for this to be a const at this point!

Reply:
100% right, fixed in [38f8dd7](https://github.com/shapeshift/web/pull/10708/commits/38f8dd73292ae6e820896de7b6841359a8d583a7)